core/main/client/browser.js

```diff
@@ -646,6 +646,16 @@ beef.browser = {
         return this.isS4() || this.isS5() || this.isS6() || this.isS7() || this.isS8();
     },
 
+    /**
+     * Return true if Epiphany
+     * @example: beef.browser.isEpi()
+     */
+    isEpi: function () {
+        // based on webkit -- therefore should look a bit like Safari
+        return window.navigator.userAgent.match(/Epiphany\//) != null;
```

Comment:
This should be based on more than just the user agent. What part of `isS()` was matching?